timemachine/cpp/src/kernels/k_nonbonded.cuh

```diff
@@ -581,14 +581,21 @@ void __global__ k_accumulate_atom_energies_to_per_mol_energies(
     per_mol_energies[idx] = energy_accumulator;
 }
 
+// k_atom_by_atom_energies is intended to be used for computing the energies of a subset of atoms
+// against all other atoms. The kernel allows changing the positions of the target atomsby passing in an
```

Comment:
atomsby -> atoms by

Reply:
Fixed typo in https://github.com/proteneer/timemachine/pull/1241/commits/aa8c0564e34e4e1f550d80e6d980d3badd456829